awx/ui_next/src/screens/CredentialType/CredentialTypeDetails/CredentialTypeDetails.jsx

```diff
@@ -82,6 +88,10 @@ function CredentialTypeDetails({ credentialType, i18n }) {
               modalTitle={i18n._(t`Delete credential type`)}
               onConfirm={deleteCredentialType}
               isDisabled={isLoading}
+              deleteDetailsRequests={deleteDetailsRequests}
+              deleteMessage={i18n._(
+                t`This credential type is currently being used by some credentials. Are you sure you want to delete it?`
```

Comment:
We are showing an alert that a credential type is being used by some credentials. But the API does not allow this action to happen. Perhaps we can handle this one differently from the others, or even in another issue. Since the user will not be able to delete the credential type.

<img width="1494" alt="image" src="https://user-images.githubusercontent.com/9053044/109542550-300ef580-7a93-11eb-97b4-d196286188bb.png">


<img width="1183" alt="image" src="https://user-images.githubusercontent.com/9053044/109541056-52077880-7a91-11eb-97b3-629615289a17.png">

Reply:
@nixocio good point.  I think the best thing to do here is, if the credential type is associated with an existing credential